projects/packages/my-jetpack/_inc/components/product-cards-section/index.tsx

```diff
@@ -90,17 +92,52 @@ const DisplayItems: FC< DisplayItemsProps > = ( { slugs } ) => {
 
 interface ProductCardsSectionProps {
 	noticeMessage: ReactNode;
+	siteIsRegistered?: boolean;
 }
 
-const ProductCardsSection: FC< ProductCardsSectionProps > = ( { noticeMessage } ) => {
-	const { ownedProducts = [], unownedProducts = [] } =
+const ProductCardsSection: FC< ProductCardsSectionProps > = ( {
+	noticeMessage,
+	siteIsRegistered,
+} ) => {
+	const { ownedProducts: inntialOwnedProducts = [], unownedProducts: inntialUnownedProducts = [] } =
 		getMyJetpackWindowInitialState( 'lifecycleStats' );
+	const [ siteOwnedProducts, setSiteOwnedProducts ] = useValueStore(
+		'siteOwnedProducts',
+		inntialOwnedProducts
+	);
+	const [ siteUnownedProducts, setSiteUnownedProducts ] = useValueStore(
+		'siteUnownedProducts',
+		inntialUnownedProducts
+	);
```

Comment:
as the updates of unowned/owned are coupled together always - we can maybe merge them into one object in ValueSture?